viewing_party/party.py

```diff
@@ -1,23 +1,188 @@
-# ------------- WAVE 1 --------------------
 
+# ------------- WAVE 1 --------------------
+# Creates a dictionary that holds movies with title, genre, and rating
 def create_movie(title, genre, rating):
-    pass
+    new_movie = {"title": title,
+                 "genre": genre,
+                 "rating": rating}
+    if title == None or genre == None or rating == None:
+        return None
+    else:
+        return new_movie
+
+# Creates a dictionary for watched movies
+
+
+def add_to_watched(user_data, movie):
+    watched_move_list = {"watched": [movie]}
+    return watched_move_list

```

Comment:
Careful here!  We don't want to accidentally overwrite the `user_data`'s "watched" list that's already been supplied to us in the tests in `test_wave_01.py`. It did pass the tests, but unfortunately the tests did not' account for "watched" lists with more than one movie in it.

Make sure we use the `user_data` instead, update it there, then return the `user_data`

```suggestion
    user_data["watched"].append(movie)
    return user_data
```